supported_versions_output.json

```diff
@@ -131,6 +131,12 @@
         "max_tracer_supported": "7.4.4",
         "auto-instrumented": false
     },
+    {
+        "integration": "crewai",
```

Comment:
Is this related to your change?

Reply:
i think a new integration was added while I was writing this PR, so its currently not in the supported versions table, though that might have changed in the last few days.